aladdin/commands/publish.py

```diff
@@ -84,8 +84,9 @@ def publish(build_only, build_publish_ecr_only, publish_helm_only):
     d = DockerCommands()
     h = Helm()
 
+    git_hash = Git.get_hash()
     # tags is a list in case we want to add other tags in the future
-    tags = [Git.get_hash()]
+    tags = [git_hash]
```

Comment:
we can likely get rid of the array here, since it's unlikely this will change soon (I think)

Reply:
agreed, I think this was here because way back in the day we used to tag with date haha (which we quickly removed)